src/upscaler.py

```diff
@@ -703,7 +706,16 @@ def run(self):
 
                 # downscale frames with Lanczos
                 Avalon.info(_('Lanczos downscaling frames'))
-                shutil.rmtree(self.extracted_frames)
+                try:
+                    shutil.rmtree(self.extracted_frames)
+                except Exception:
+                    if platform.system() == 'Windows':
+                        for file in self.extracted_frames.iterdir():
+                            Avalon.info(_("Remove file:[%s] readOnly attribute." % file))
+                            os.chmod(file, stat.S_IWRITE)
```

Comment:
This is a file object operation, so it should be handled by the file object.
Use `extracted_frame.chmod`.